projects/knora-ui/src/lib/viewer/values/int-value/int-value.component.spec.ts

```diff
@@ -106,9 +109,7 @@ describe('IntValueComponent', () => {
 
       expect(testHostComponent.inputValueComponent.mode).toEqual('read');
 
-      expect(valueInputNativeElement.value).toEqual('1');
-
-      expect(valueInputNativeElement.readOnly).toEqual(true);
+      expect(valueReadModeNativeElement.outerText).toEqual('1');
```

Comment:
What is the difference between outer and inner text?

Reply:
https://stackoverflow.com/questions/18481382/what-is-the-difference-between-innertext-and-outertext

My noobiness :) I initially wrote outText for all the tests then realized I should be using innerText and missed a couple of them. 

switched to innerText in 37865b7

Reply:
great, thx.